torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9560,14 +9530,20 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            skips=(
+               # Float did not match double
+               DecorateInfo(unittest.expectedFailure, 'TestGradients', 'test_fn_grad'),
```

Comment:
so gradients are all wrong here?

Reply:
As best the test suite can tell: yes

Reply:
Nice. Is there a hi pri issue about this, or we don't care about `cov` (or whatever this function is)?